cppcache/integration-test/CMakeLists.txt

```diff
@@ -30,6 +30,7 @@ target_link_libraries(test-cppcache-utils
   PUBLIC
     apache-geode
     framework
+    SPDLog::SPDLog
```

Comment:
Why are the integration test dependent on the `SPDLog` library? It's use should be hidden at the integration level. The library should be statically compiled into the `apache-geode` library.

Reply:
This may or may not be vestigial - I'll have to check.  spdlog is a header-only library, at least as we're using it, so this dependency just means we need to include one of its headers.  But again, I'm not sure that's the case any more, so I will try to remove this ref.

Reply:
removed